examples/.markdownlint.yaml

```diff
@@ -0,0 +1,274 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
How is this going to be used in CI? Will markdownlint use the first yaml it finds going up the directory tree from a file?

Reply:
Actually, the CI will just look at the top level yaml. To run the yaml from examples, one needs to explicitly run markdownlint in examples folder. I'd say, to limit the scope, let's consider introducing that to CI in a separate PR.